DS13/code/modules/section31/datum_section31.dm

```diff
@@ -0,0 +1,376 @@
+#define SECTION31_HUMAN "human"
+#define SECTION31_AI "AI"
+
+/datum/antagonist/section31
+	name = "Section 31 Agent"
+	roundend_category = "Section 31 Agents"
+	antagpanel_category = "Section 31 Agent"
+	job_rank = ROLE_SECTION31
+	antag_moodlet = /datum/mood_event/focused
+	var/special_role = ROLE_SECTION31
+	var/employer = "Section 31"
+	var/give_objectives = TRUE
+	var/should_give_codewords = TRUE
+	var/should_equip = TRUE
+	var/section31_outfit = /datum/outfit/section31
+	var/section31_kind = SECTION31_HUMAN //Set on initial assignment
+	can_hijack = HIJACK_HIJACKER
+
+/datum/antagonist/section31/on_gain()
+	if(owner.current && isAI(owner.current))
+		section31_kind = SECTION31_AI
+
+	SSticker.mode.traitors += owner
+	owner.special_role = special_role
+	if(give_objectives)
+		forge_section31_objectives()
+	equip_op()
+	finalize_section31()
+	..()
+
+/datum/antagonist/section31/apply_innate_effects()
+	if(owner.assigned_role == "Clown")
+		var/mob/living/carbon/human/section31_mob = owner.current
+		if(section31_mob && istype(section31_mob))
+			if(!silent)
+				to_chat(section31_mob, "Your training has allowed you to overcome your clownish nature, allowing you to wield weapons without harming yourself.")
+			section31_mob.dna.remove_mutation(CLOWNMUT)
+
+/datum/antagonist/section31/remove_innate_effects()
+	if(owner.assigned_role == "Clown")
+		var/mob/living/carbon/human/section31_mob = owner.current
+		if(section31_mob && istype(section31_mob))
+			section31_mob.dna.add_mutation(CLOWNMUT)
```

Comment:
Perhaps the `owner` should be notified that they'll injure themselves from now on?

Reply:
No clue what that does but grabbed it from tatorcode in default tg from this fork, it was the closest thing to a Section 31 agent that KMC could point me towards in reference to code; I can remove that part though